config/DefaultFlags.cmake

```diff
@@ -0,0 +1,54 @@
+if(CMAKE_Fortran_COMPILER_ID STREQUAL "GNU")
+  set(
+    CMAKE_Fortran_FLAGS_INIT
+    "-fimplicit-none"
+    "-ffree-line-length-132"
+    "-fno-range-check"
+  )
+  set(
+    CMAKE_Fortran_FLAGS_RELEASE_INIT
+  )
+  set(
+    CMAKE_Fortran_FLAGS_DEBUG_INIT
+    "-Wall"
+    "-Wextra"
+    "-Wimplicit-procedure"
+    "-std=f2018"
+  )
+elseif(CMAKE_Fortran_COMPILER_ID MATCHES "^Intel")
+  set(
+    CMAKE_Fortran_FLAGS_INIT
+  )
+  set(
+    CMAKE_Fortran_FLAGS_RELEASE_INIT
+  )
+  if(WIN32)
+    set(
+      CMAKE_Fortran_FLAGS_DEBUG_INIT
+      "/stand:f18"
+      "/warn:declarations,general,usage,interfaces,unused"
+    )
+  else()
+    set(
+      CMAKE_Fortran_FLAGS_RELEASE_INIT
+    )
```

Comment:
Is their a reason non-Windows OS has this, but WIN32 does?

Reply:
```suggestion
```